vault-core/src/main/java/org/apache/jackrabbit/vault/util/RepositoryCopier.java

```diff
@@ -410,11 +410,17 @@ private void copy(AutoSave autoSave, Node src, Node dstParent, String dstName, b
                             currentSize+=s;
                         }
                     } else {
-                        Value v = p.getValue();
-                        dst.setProperty(pName, v);
-                        long s= p.getLength();
-                        totalSize+=s;
-                        currentSize+=s;
+                        // Filter out unwanted properties.  Remove them from the destination if they are excluded by filter
```

Comment:
The same logic would need to be applied to `sysCopy` which uses JCR System View to copy nodes from src to dst. This requires some more code changes as you probably need to implement a filtering ContentHandler (based on http://www.saxproject.org/apidoc/org/xml/sax/helpers/XMLFilterImpl.html).

Reply:
Thanks @kwin does the current sysCopy method already filter out nodes somehow, but it is missing property based filtering?  I'm a little lost looking at the current [sysCopy](https://github.com/apache/jackrabbit-filevault/blob/master/vault-core/src/main/java/org/apache/jackrabbit/vault/util/RepositoryCopier.java#L505) having never been in here before.

Reply:
@kwin it's not clear to me how we would intercept the ContentHandler found in syscopy, which is an ImportHandler defined in oak-run.   A new implementation would, perhaps, need to extend ImportHandler to apply filtering, and the pass it on to the exportSystemView call as done already.

If you have any guidance, or examples, that would be greatly appreciated.  If you'd like to schedule some time to discuss this, I'd welcome it.